gpflow/config/__config__.py

```diff
@@ -1,106 +1,142 @@
-import os
 import contextlib
+import enum
+import os
+from dataclasses import dataclass, field, replace
+from typing import Optional
 
 import numpy as np
 import tabulate
 import tensorflow as tf
 
 __all__ = [
```

Comment:
A little example on how to use this config would be great! 
- how to get the default value
- how to change it

Reply:
Do you mean the documentation for the `config.py`?